### DISCUSSION TITLE
Direction?

### DISCUSSION THREAD

**Discussion by 0xRenegade** [devkennyy/rungeon#99] [Ideas]
Where is the direction you want this repo to head? Defining a roadmap or project can help contributors.

For example, an issue has been opened for a backend and a draft pull request (at the time of writing this post), but what direction is the application going? More in-depth story for a longer-run(geon) text-based adventure game or possibly even a text-based adventure game builder with components created in a modular way?

Thoughts?

**Comment by Shabbyconnor** [comment]
And how text-based are we talking about? I was thinking about adding some sort of visual puzzle, but I can hold off if it doesn't fit the direction of the project.

**Comment by devkennyy** [comment]
My goal was to have lots of people contributing stages and the project would become a huge library of puzzles. I've removed the text-adventure from the description of the project as that isn't a focus, it is merely a tool. My initial goal remains the same, we just have to find ways to make the project very easy to contribute to. In interactive builder for stages seems viable in the foreseeable future. I hope I've made myself clear but the project is definitely subject to change if that's what is necessary.

**Comment by gruelingpine185** [comment]
Can we add more labels for issues to better organize them? I'm thinking of `ui`, `mobile`, `desktop`, and `backend`. More may come to mind later on, but I think it'll be helpful to day the lease.

**Reply by devkennyy** [reply]
Absolutely!
